app-text/calibre/calibre-7.8.0.ebuild

```diff
@@ -0,0 +1,242 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
```

Comment:
This commit message says it is adding 7.8.0, but the actual effect of the commit includes *two* changes.

One is adding 7.8.0, the other is upgrading PYTHON_COMPAT.

```diff
commit 1b0c4b54f27e0efaa2dee96b7ee47fa9946d0faa
Author:     Denis Pronin <dannftk@yandex.ru>
AuthorDate: Sat Apr 6 01:54:58 2024
Commit:     Denis Pronin <dannftk@yandex.ru>
CommitDate: Sat Apr 6 01:54:58 2024

    app-text/calibre: add calibre-7.8.0
    
    Signed-off-by: Denis Pronin <dannftk@yandex.ru>

diff --git a/app-text/calibre/Manifest b/app-text/calibre/Manifest
index db213d8ba916..c8c64b303761 100644
--- a/app-text/calibre/Manifest
+++ b/app-text/calibre/Manifest
@@ -6,3 +6,5 @@ DIST calibre-7.6.0.tar.xz 43929700 BLAKE2B a529bf32511f8927683d6b89962a2e0e1ca6f
 DIST calibre-7.6.0.tar.xz.sig 566 BLAKE2B c7fac255eb3ea49aead5780cc8089be253e551d729bb80c3f59bdb269e05e603557dbe8967175a6bb49e88d30673c89b2217c18757b718dba8e5b7d3eb21330c SHA512 a9af6200b27263333e2d3c54e4aea7f7664b6c4d3ba8cdd5958c9eeaf0c7ee4ba7465df57fd526064c44a9d093b246a64ab94819c6235030162f30f7f570b403
 DIST calibre-7.7.0.tar.xz 44006620 BLAKE2B f29dcf4c6550c7649dfddcaa4f5390ebf23ae7ef8d8103c9d001c9fcb8ca4a120c7cc8d5c2c36acced6672d78e45b6a93c6c505482f78e21ecc170c56c2641b3 SHA512 8444811e63be007e075b716c70c7df989408b8a323068fb763a07b27a2c52557e5625f311ac7672d5367df9dd3d071354cf3dd1102a0aa057e5cbbdc09ed24df
 DIST calibre-7.7.0.tar.xz.sig 566 BLAKE2B 6e21d5e8517abd2cf9e278cf40308709422b97cf2c953e1f1b8487aeb6561c57902d233fb397abfe50b8a42ce28071426477171a9d5b0b055af5cf41fdb78ef4 SHA512 17d91e46aa0dc3e12f1d65c5956f3c0f6b9fb07705042efd73508cc0040d0df2bae939544fd4bc123064393174057b5107cb38167b2db2f9477988ce369e10ac
+DIST calibre-7.8.0.tar.xz 44167208 BLAKE2B 676f5519b2809ca91c972ce0b4227aeac977945248063be58f8ed387c2dc1285f01e0d59c52ba9dea3a25842be1f97cdaece7f1eac5c208503240ff9ac3f0093 SHA512 c90d247af3a8ba3b2a94ca3a03bf9ee4d1c8f07d6feaa1ec045d639f448c476c4be9568e66845e7067e32b746e0ab034be65084a60f2b64f961f6678a30910e7
+DIST calibre-7.8.0.tar.xz.sig 566 BLAKE2B 03f283bee606d3071a2602ad2b1cc962b3d0cf6737ca294955524da6de7f22ddbe740d9b455332961b847331ad15f47efa884cc65bfe1a42eb46bef9f7a4a1dd SHA512 527a931f6d8701580b7b405124f53b54bb6ca65056aa41d677806a00c2ece3cb138c1c37e0d29394ee4e83487dec9dc7e82b6db63a4154e19f8463c19956e040
diff --git a/app-text/calibre/calibre-7.7.0.ebuild b/app-text/calibre/calibre-7.8.0.ebuild
similarity index 99%
copy from app-text/calibre/calibre-7.7.0.ebuild
copy to app-text/calibre/calibre-7.8.0.ebuild
index fc786f35d76c..ee964558cb0d 100644
--- a/app-text/calibre/calibre-7.7.0.ebuild
+++ b/app-text/calibre/calibre-7.8.0.ebuild
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{10..11} )
+PYTHON_COMPAT=( python3_{10..12} )
 PYTHON_REQ_USE="sqlite,ssl"
 
 inherit edo toolchain-funcs python-single-r1 qmake-utils verify-sig xdg
```

And as it happens, such an unannounced change also breaks the repository CI, since python 3.12 cannot be added yet.

This is why it is important to describe each change you make, in separate fine-grained commits for reliability when reviewing.